suite-native/module-accounts-import/src/screens/XpubScanScreen.tsx

```diff
@@ -92,7 +92,7 @@ export const XpubScanScreen = ({
     };
 
     return (
-        <Screen header={<AccountImportHeader activeStep={1} />}>
+        <Screen header={<AccountImportHeader origin={route.params.origin} activeStep={1} />}>
```

Comment:
This will fail when the app is in "onboarding mode" - like a first run of the app.